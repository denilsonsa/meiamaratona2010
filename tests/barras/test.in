66
10101101110110001010111100010110100011011110101010101110011001101011100111001010010001110100101
10101101110110001010111100010110100011011110100010111110011001101111100111001000010001110100101
10100101110001001010011100111010110011001110101010101111011000101101000111101010001101110110101
10101101110001011011110101011110111101011110101010100001010100001110010100100011100101110100101
10101101110111011011101101101110001101011101101010111001011100101110100100111010011101101100101
10101000110101111000110101111010010011000101101010100010010011101000010100111010011101000100101
10100010110110001001001100100110110111000110101010101000011001101001110100100010010001110010101
10101011110110001000110101000110111011001001101010101000011100101110010100010010000101001110101
10100110010111101011101101111010110001010001101010100111010011101001110100100011101001000100101
10101111010110111001100101100010011001011101101010111010011001101001000111001010100001010000101
10100011010001101010001100110010001011001001101010110011010100001000010111010011101001000100101
10100110010001101011000100010110101111000101101010101000010010001001000101110010001001110010101
10100010110111101011000101100010001101011000101010100100011100101101100110011010100001000010101
01000101001001001011001101010000100001011011011000011000110100101010100111010010100001000101011
01001010100110110100110110110100010011111001000000100011010110110001011010110011001101110100111
11100100100101101101001110100000001101111011110100100111101010011010001010001000010100010001110
01110000011100101011110110111111110101101101100010010011001011101001011000100110110110011011111
00111000011001010001011100110110000000111011111011010001000001000110110100101011101010010000010
11100101111000001111001110111001111011110000011011101111000000000100110101101101000100011100111
11000100111010001000010100101010011000111111111010000000101000001011100010110011110010001111010
01000001101100011001111100100000010010100111111101111101010001110010110010000011110001001011010
01111100110101100101100010000011110010010000110010011101000110100110010000100001100001000001000
01101000011100111111010111101101011100101010101110100110000101010101100001011000011011001001010
10100110110100001001101100010010011101001000101010110100010001101001100101100010001101101000101
10100110110010111010011101001110010001010011101010110111010011001111010110001010110001100100101
10101001110010001001011101110010000101001110101010100110011101101000110110111010110001011110101
10101100110000101011100100001010110011001101101010111011011000101000110111011011101101110110101
10100100010111001001000100010010100001001011101010101100011010001100010100011011110101111010101
10101001110010111010000101000010011011001101101010101100011001001100100110111011101101110110101
10101110010010001011100100110110000101000010101010111011011101101100100101111011101101111010101
10101100110000101001011101100110100111001000101010110010010111101011000110001011001001111010101
10100110110010001010011101001110110011001011101010101100011011101000110110001011000101011000101
10100111010010111010000100101110111001010011101010100110010001101111010100110010011001011000101
10101101110001011011101101100010001011000110101000100001011011001000010110110010010001000100101
10101000110100011001001100110010001011011000101010111001011011001010000100100010000101010000100
10001110110001101011011101100010111101001100101010101000010010001110100100111011001101100110101
10100110010010011001100100011010110111001001101010100100010001001000100100111011101001010000101
10101101110111011001100101011110110001011101101010100100010010001110100110110011011001000100101
10101101110111101011011101110110010011000110101010100010010111001011100111010011100101001000101
10100011010001101001001100100110110001011101101010111001011011001110010110011010010001001000101
10100100110010011000101101100010101111011101101010110011010100001110010111001010001001001000101
10101110110111011000101101100010111011011101101010100001011101001010000111001010111001101100101
10100011010101111010001101101110010011000101101010111010010000101001000100100010000101011100101
10101110110001101000101101000110010011010111101010100001010000101110100110011010001001010000101
10101011110010011001100100011010110111001100101010110011011100101011100100001011100101001110101
10100011010100011001001100100110111011011011101010100100010001001011100101000010100001110100101
00000100101101001110100110100111000100001101011011100111000110011011010001010000000010100010010
10111001010111101101110101010011100101011001001010001101000100010011000100110111001111110100000
10111101000000111100101001001000001011111101000010100100101000100010110010111110110010000100010
01010110111110011111010011111111101001111100000011101000010111001011101100101001010000000100100
00100010100110011000011110110100001101111010100011010000111011100000000000101100011100001110100
00011011111010100000011001110010000111100100000101111010011011111011010100110101111100111110000
11011011101001010001011001110110101111010010100110101110111010101001011011101110100101111010000
00101010110110111001011100111011101010100000111001011110011111111001100001101001001001011000001
11111000100100010000101011100111000110100111011011110001101011100011011101001111110001000011001
10101101101010010001101011000001100101110111010010111110011111010011101101000001101001100011110
10100111010010001001110100010010010001001000101010100110010001101011110111011011101101000110101
10100001010100001011100100100010111001001011101010100110011011101011000111101011101101101110101
10101000010001001001110100001010110011001000101010101111010011001001100110100010111101110110101
10100101110010111001110101100110001001000100101010110100010001101100010111011011011101111010101
10101000010001001011100100111010100111010011101010110100010011001110110110010011101101100010101
10100101110100111001011101001110111001010000101010111101011101101100100100110010011001101000101
10100001010011011000100101110010100001010011101010110010010001101011110101111011011101111010101
10101000010010111011100100101110010111010011101010110010011101101110110110111010011001110110101
10100100010100001001110101001110100111011100101010110111011101101101110101111010011001110110101
10101001110100001011001100100010111001011100101010110010010110001111010110100010111101101000101
