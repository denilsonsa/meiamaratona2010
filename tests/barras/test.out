856943414089
XXXXXXXXXXXX
856943414089
893633360809
877807009552
460329753557
952280615880
650472600735
137354555897
381517918066
004192163997
105969688470
935505802163
XXXXXXXXXXXX
XXXXXXXXXXXX
XXXXXXXXXXXX
XXXXXXXXXXXX
XXXXXXXXXXXX
XXXXXXXXXXXX
XXXXXXXXXXXX
XXXXXXXXXXXX
XXXXXXXXXXXX
XXXXXXXXXXXX
950159748232
204617070092
307581465970
888548216561
665490938757
887220223390
683288662575
624032701961
044570910072
011651059394
XXXXXXXXXXXX
XXXXXXXXXXXX
XXXXXXXXXXXX
121082877596
871657889227
838720744908
002257020188
229567160078
779577396042
064829938834
709426339176
621081104305
042278874669
XXXXXXXXXXXX
XXXXXXXXXXXX
XXXXXXXXXXXX
XXXXXXXXXXXX
XXXXXXXXXXXX
XXXXXXXXXXXX
XXXXXXXXXXXX
XXXXXXXXXXXX
XXXXXXXXXXXX
XXXXXXXXXXXX
588351778474
786071957536
839113716483
678459881499
482819004583
911286350909
673352035826
817882099593
813787500437
939602557130
